sky/core.py

```diff
@@ -139,20 +140,22 @@ def cost_report() -> List[Dict[str, Any]]:
         A list of dicts, with each dict containing the cost information of a
         cluster.
     """
-    cluster_reports = global_user_state.get_clusters_from_history()
 
-    def get_total_cost(cluster_report: dict) -> float:
-        duration = cluster_report['duration']
-        launched_nodes = cluster_report['num_nodes']
-        launched_resources = cluster_report['resources']
-
-        cost = (launched_resources.get_cost(duration) * launched_nodes)
-        return cost
+    # aggregate records for spot controller
+    if cluster_name is not None:
+        cluster_reports = cost_utils.aggregate_all_records(verbose=False)
```

Comment:
I feel like whether to aggregate the records are orthogonal to whether the `cluster_name` is provided. For example, we can still output the reports without aggregation, when the `cluster_name` is provided. Is it possible to change the API to `cost_report(cluster_names: Optional[List[str]], aggregate_by_cluster_name: bool)`? I also changed the `cluster_name` to be a list of cluster_names so that the user can provide multiple clusters, just as what we did for the `core.status`